### PR TITLE
Fixing old issues (before project)

### PR DESCRIPTION
**Pull request by mokot**

### CODE REVIEW

File: src/entity/user.entity.ts [hunk 1]
```diff
@@ -83,4 +74,10 @@ export class User {
   @CreateDateColumn()
   updated_at: Date;
 
+  @BeforeInsert()
+  nameToLowerCase() {
+    this.first_name = this.first_name.toLowerCase();
+    this.last_name = this.last_name.toLowerCase();
```
Comment (jakapurg) [inline src/entity/user.entity.ts:80]: Is this necessary?

File: src/module/team/team.controller.ts [hunk 2]
```diff
@@ -41,7 +41,11 @@ export class TeamController {
   @UseGuards(JwtAuthGuard, RoleGuard)
   @Get('all')
   async getAll(): Promise<Team[]> {
-    return await this.teamService.getAll();
+    return this.teamService.getAllBy(
+      {
+        relations: ['admin']
+      }
+    );
```
Comment (jakapurg) [inline src/module/team/team.controller.ts:48]: You should probably return members of the team.

File: src/module/team/team.service.ts [hunk 3]
```diff
@@ -47,13 +35,17 @@ export class TeamService {
    * @returns 
    */
   @Transactional()
-  async getAllUsersInTeam(teamId: number): Promise<User[]> {
-    const team = await this.getTeamByTeamId(teamId);
+  async getAllUsersInTeam(id: number): Promise<User[]> {
+    const team = await this.getTeamBy(
+      { where: { id: id } }
+    );
```
Comment (jakapurg) [inline src/module/team/team.service.ts:41]: Is this function really needed? You should return team users when getting team

File: src/module/team/team.controller.ts (hunk 2 above)
Comment (mokot) [reply src/module/team/team.controller.ts:54]: This call returns all teams and its admin users.

Comment (jakapurg) [reply src/module/team/team.controller.ts:54]: Yeah, but it does not return members of the team. Field users on team

**Review by jakapurg — APPROVED** [review]

**Pull request merged**